src/swap/sideshift.js

```diff
@@ -50,15 +47,15 @@ type FixedQuote = {
 
 type FixedQuoteRequestParams = {
   depositMethod: string,
-  settlMethod: string,
+  settleMethod: string,
   depositAmount: string
 }
 
 type OrderRequest = {
   createdAt: string,
-  createdAtISO: string,
-  expiresAt: Date,
-  expiresAtIso: string,
+  createdAtISO: Date,
+  expiresAt: string,
+  expiresAtIso: Date,
```

Comment:
`expiresAtISO`

Reply:
And it's a string. Can't transmit `Date` across `JSON` unless you have some witchcraft